packages/cli/generators/controller/index.js

```diff
@@ -133,18 +132,26 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
 
     if (_.isEmpty(modelList)) {
       return this.exit(
-        `No models found in ${this.artifactInfo.modelDir}.
-        ${chalk.yellow(
-          'Please visit http://loopback.io/doc/en/lb4/Controller-generator.html for information on how models are discovered',
-        )}`,
+        chalk.yellow(
```

Comment:
![image](https://user-images.githubusercontent.com/50331796/79911358-31d94880-83ee-11ea-9f33-2d50afb36b5a.png)

Reply:
Not sure if it matters but before only the "Please visit..." portion was in yellow and now all of it is

Reply:
<img width="889" alt="截屏2020-04-22上午9 23 29" src="https://user-images.githubusercontent.com/50331796/79987063-f0db4580-847a-11ea-80cd-fe86cf479b3d.png">
I modified it so that it is the same as before.